src/Persistence_matrix/include/gudhi/Persistence_matrix/columns/entry_types.h

```diff
@@ -0,0 +1,327 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hannah Schreiber
+ *
+ *    Copyright (C) 2022-24 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+/**
+ * @file entry_types.h
+ * @author Hannah Schreiber
+ * @brief Contains the @ref Gudhi::persistence_matrix::Entry, @ref Gudhi::persistence_matrix::Entry_column_index and
+ * @ref Gudhi::persistence_matrix::Entry_field_element classes, as well as the
+ * @ref Gudhi::persistence_matrix::Dummy_entry_column_index_mixin and
+ * @ref Gudhi::persistence_matrix::Dummy_entry_field_element_mixin structures.
+ * Also defines the std::hash method for @ref Gudhi::persistence_matrix::Entry.
+ */
+
+#ifndef PM_MATRIX_CELL_H
+#define PM_MATRIX_CELL_H
```

Comment:
Maybe rename also the directives:
```suggestion
#ifndef PM_MATRIX_ENTRY_H
#define PM_MATRIX_ENTRY_H
```